xml/System.Net.WebSockets/ValueWebSocketReceiveResult.xml

```diff
@@ -49,10 +49,10 @@
         <Parameter Name="endOfMessage" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="count">To be added.</param>
-        <param name="messageType">To be added.</param>
-        <param name="endOfMessage">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="count">The number of bytes received.</param>
+        <param name="messageType">The type of message that was received.</param>
+        <param name="endOfMessage">Whether this is the final message.</param>
+        <summary>Creates an instance of the <see cref="T:System.Net.WebSockets.ValueWebSocketReceiveResult" /> value type.</summary>
         <remarks>To be added.</remarks>
```

Comment:
Those elements were autogenerated by the Docs team, I don't think we should remove them. The "To be added" string is what they consider "not documented". Since remarks are optional, they can stay with that value. They won't show up among the undoc APIs in our stats.